tools/istio-iptables/pkg/validation/vld_stub.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !aix,!darwin,!dragonfly,!freebsd,!linux,!netbsd,!openbsd,!solaris
```

Comment:
What is the point of having this? It seems useless to compile the binary on these platforms, as it will just fail at runtime?

Reply:
It returns the errors to the callers to these functions at runtime and leaves the decision to them for what to do at runtime, instead of failing at the compile time.

Reply:
Or you might prefer a failure at the compile-time, which stops those platforms from building istio?

Reply:
I would prefer a compile time error. That is what the compiler is for :slightly_smiling_face:

Reply:
Done.